ui/src/app/kvm/views/KVMDetails/CoreResources/CoreResources.tsx

```diff
@@ -23,6 +35,24 @@ const CoreResources = ({ cores, dynamicLayout }: Props): JSX.Element => {
       <div className="core-resources__meter">
         <PodMeter allocated={cores.allocated} free={cores.free} segmented />
       </div>
+      <div>
+        {noPin && noAvailable ? (
```

Comment:
You can use short circuit evaluation (`&&`) if you only want to render the truthy option, e.g.
``` jsx
{!(noPin && noAvailable) && (
  <div>
    ...
```
In general I prefer to keep variables in the "positive" to avoid double negatives, which is why I've suggested `showPinnedSection` above. i.e. it's much easier to understand "show" and "not show" instead of "not show" and "not not show".
``` jsx
{showPinnedSection && (
  <div>
    ...
}